src/protocol.cpp

```diff
@@ -2772,6 +2772,12 @@ bool CProtocol::GetStringFromStream ( const CVector<uint8_t>& vecIn, int& iPos,
     return false; // no error
 }
 
+QLocale::Country CProtocol::GetCountryFromStream ( const CVector<uint8_t>& vecIn, int& iPos )
```

Comment:
Mmm, I'd leave this as the 2 byte value until it's been through `CLocale::WireFormatCountryToQtCountry`.  Turning it into a _current_ Qt QLocale::Country isn't right: it's always a Qt 5 (or less) QLocale, so it's invalid if, say, Qt8 changes QLocale::Country into something the `static_cast<QLocale::Country> ( GetValFromStream ( vecIn, iPos, 2 ) )` would fail for.

I can't see the point of doing half the work here and half the work there.

Here, just get the ushort value off the stream and pass it to CLocale::WireFormatCountryToQtCountry to get the country.

Reply:
Um, thanks for that suggestion. I have no idea why I did it the way I did initially. I fully agree with what you said.

I've now
- renamed `CLocale::WireFormatCountryToQtCountry` to `CLocale::WireFormatCountry`**`Code`**`ToQtCountry` and updated it to return an `unsigned short` instead of `QLocale::Country`
- removed the obsolete, redundant casting in `CProtocol::GetCountryFromStream`

Can you re-check? Is `us` the proper prefix for `unsigned short`? `usCountry...` has potential for confusion, but I've got no idea what else to call it.

Reply:
Yeah, we don't use a lot of unsigned shorts...  And I agree, `usCountry` is not nice.  `wfCountry` (wire-format)? :)

Reply:
I've ran a quick grep over the current code and the `i` prefix is used for anything-int, independent of signedness or width. Therefore, I've reverted this back to `iCountryCode` for now.